pydantic_ai_slim/pydantic_ai/_cli.py

```diff
@@ -119,6 +119,24 @@ def cli(  # noqa: C901
 """,
         formatter_class=argparse.RawTextHelpFormatter,
     )
+
+    subparsers = parser.add_subparsers(dest='command', help='Available commands')
+
+    # Web subcommand (only available for clai)
+    if prog_name == 'clai':
+        web_parser = subparsers.add_parser('web', help='Launch web chat UI for discovered agents')
+        web_parser.add_argument('--host', default='127.0.0.1', help='Host to bind the server to (default: 127.0.0.1)')
+        web_parser.add_argument('--port', type=int, default=8000, help='Port to bind the server to (default: 8000)')
+        web_parser.add_argument('--dir', type=Path, help='Directory to search for agents (default: current directory)')
+        web_parser.add_argument(
+            '--config', type=Path, help='Path to agent_options.py config file (overrides auto-discovery)'
```

Comment:
TODO adding a URL to a gist or doc of the default file would help users more easily customize their setup

Reply:
referenced the default `agent_options.py` in the clai README.md